practice-app/app/views.py

```diff
@@ -59,4 +60,41 @@ def WorldTime():
     else:
         return render_template("worldtime.html")
 
+def get_game_id_from_name(game_name):
+    steam_api_game_list_url = "https://api.steampowered.com/ISteamApps/GetAppList/v2/"
+    response = requests.get(steam_api_game_list_url).json()
+    all_games_list = response["applist"]["apps"]
+    game_name = "".join(game_name.lower().split())
+    for game in all_games_list:
+        if game_name == "".join(game["name"].lower().split()): 
+            return game["appid"]
+    return -1
 
+def get_game_information_from_id(game_id): 
+    steam_api_get_game_url = "https://store.steampowered.com/api/appdetails?appids={game_id}"
+    response = requests.get(steam_api_get_game_url.format(game_id=game_id)).json()
+    game_information = response[str(game_id)]["data"]
+    return game_information
+
+def get_game_information_from_name(game_name): 
+    game_id = get_game_id_from_name(game_name)
+    if game_id == -1: 
+        return None
+    game_information = get_game_information_from_id(game_id)
+    return game_information
+
+@app.route("/get_game_information", methods= ["GET", "POST"])
+def get_game_information():
```

Comment:
How about moving all relevant functions to a separate file for better structure? As there will be many more implementations on our codebase, safe to say seperating them will be a better choice.

Reply:
Should the new seperated file be used from everyone or should it be special to the game information API implementations? Also should I also move the @app.route function too?